generators/app/templates/_dockerTaskGeneric.ps1

```diff
@@ -41,9 +41,9 @@ $imageName="<%= imageName %>"
 $projectName="<%= composeProjectName %>"<% if (includeStartDebugging) { %>
 $serviceName="<%= serviceName %>"
 $containerName="<%= '${projectName}_${serviceName}' %>_1"<% } %>
+$isWebProject=$<%= isWebProject %><% if (isWebProject) { %>
 $publicPort=<%= portNumber %>
-$isWebProject=$<%= isWebProject %>
-$url="http://docker:$publicPort"<% if (projectType === 'dotnet' && (dotnetVersion === 'RC2' || dotnetVersion === 'RTM')) { %>
```

Comment:
>$url [](start = 0, length = 4)

This is still being referenced below. If you're going to if out web stuff for the console case you need to get it all.